Dataset.cpp

```diff
@@ -431,14 +432,16 @@ void Dataset::DatasetRender(const Box &alignedRegion, AmrPicture *apptr,
   XmStringFree(sNewLevel);
   
   sprintf(minInfoV, fstring, rMin);
-  sprintf(minInfo, "Min:%s", minInfoV);
-  XmString sNewMin = XmStringCreateSimple(minInfo);
+  std::string minInfo("Min:");
```

Comment:
I did not change this for compatibility, but we could add a space here:
```suggestion
  std::string minInfo("Min: ");
```